ydb/docs/en/core/development/load-actors-storage.md

```diff
@@ -79,4 +104,31 @@ StorageLoad: {
 
 When viewing test results, the following value should be of most interest to you:
 
-* ` ReadSpeed@ 100%`: 100 percentile of read speed in MB/s, e.g., `60.86`.
+* `ReadSpeed@ 100%`: 100 percentile of read speed in MB/s, e.g., `60.86`.
+
+### Read only {#readonly}
+
+Before the start of the main load cycle the `1 GB` data block of blobs with sizes between `1 MB` and `5 MB` is allocated. To avoid overloading the system with write requests the number of simultaneously processed requests is limited by the value of `5`. After completing the initial data aloocation the main cycle is launched, which consists of read requests sent with increasing rate: from `10` to `50` requests per second, the rate will increase gradually for `300` seconds.
```

Comment:
```suggestion
Before the start of the main load cycle the `1 GB` data block of blobs with sizes between `1 MB` and `5 MB` is allocated. To avoid overloading the system with write requests the number of simultaneously processed requests is limited by the value of `5`. After completing the initial data allocation the main cycle is launched. It consists of read requests sent with increasing rate: from `10` to `50` requests per second, the rate will increase gradually for `300` seconds.
```